bpf/bpf_lxc.c

```diff
@@ -679,6 +690,17 @@ static __always_inline int handle_ipv4_from_lxc(struct __ctx_buff *ctx,
 		}
 	}
 
+#if defined(ENABLE_HOST_FIREWALL) && !defined(ENABLE_ROUTING)
+	/* If the destination is the local host and per-endpoint routes are
+	 * enabled, jump to the bpf_host program to enforce ingress host policies.
+	 */
+	if (*dstID == HOST_ID) {
```

Comment:
I think this check works only if `--enable-remote-node-identity=true`. Otherwise, we cannot distinguish between local a node (`HOST_ID`) and a remote node (`REMOTE_HOST_ID`).

Reply:
I'll restrict `enable-host-firewall` + `enable-endpoint-routes` on `enable-remote-node-identity` in a follow up.